From 4f36a99bd8215246761bd7d9261614fbf70a7cb4 Mon Sep 17 00:00:00 2001
From: Sergey Kolekonov <skolekonov@mirantis.com>
Date: Mon, 6 Apr 2015 16:28:52 +0300
Subject: [PATCH] requirements.txt: adjust for Ubuntu 14.04

* Routes!=2.0 restriction is necessary only for running tests with python 3.x,
  routes 2.0 work just fine with python 2.7
* argparse is not required as it's provided by python
* Remove upper bound for netaddr, because MOS mirrors contains newer version
  (0.7.14)
---
 requirements.txt | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index d597cdc..d78a269 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -5,9 +5,8 @@ pbr>=0.6,!=0.7,<1.0
 
 Paste<=1.7.5.1
 PasteDeploy>=1.5.0,<=1.5.2
-Routes>=1.12.3,!=2.0,<=2.1
+Routes>=1.12.3,<=2.1
 anyjson>=0.3.3,<=0.3.3
-argparse
 Babel>=1.3,<=1.3
 eventlet>=0.15.1,<=0.15.2
 greenlet>=0.3.2,<=0.4.2
@@ -17,7 +16,7 @@ iso8601>=0.1.9,<=0.1.10
 jsonrpclib<=0.1.3
 Jinja2<=2.7.2
 keystonemiddleware>=1.0.0,<1.4.0
-netaddr>=0.7.12,<=0.7.13
+netaddr>=0.7.12
 python-neutronclient>=2.3.6,<3
 SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
 WebOb>=1.2.3,<=1.3.1
-- 
1.9.1

